modules/serverless-queue-proxy-metrics.adoc

```diff
@@ -0,0 +1,44 @@
+[id="serverless-queue-proxy-metrics_{context}"]
+= Queue proxy metrics
+
+Each Knative service has a proxy container that proxies the connections to the application container. A number of metrics are reported for the queue proxy performance.
+
+You can use the following metrics to measure if requests are queued at the proxy side and the actual delay in serving requests at the application side.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`revision_request_count`
+|The number of requests that are routed to `queue-proxy` pod.
+|Counter
+|`configuration_name`, `container_name`, `namespace_name`, `pod_name`, `response_code`, `response_code_class`, `revision_name`, `service_name`
+|Integer (no units)
+
+|`revision_request_latencies`
+|The response time of revision requests.
+|Histogram
+|`configuration_name`, `container_name`, `namespace_name`, `pod_name`, `response_code`, `response_code_class`, `revision_name`, `service_name`
+|Milliseconds
+
+|`revision_app_request_count`
+|The number of requests that are routed to the `user-container` pod.
+|Counter
+|`configuration_name`, `container_name`, `namespace_name`, `pod_name`, `response_code`, `response_code_class`, `revision_name`, `service_name`
+|Integer (no units)
+
+|`revision_app_request_latencies`
+|The response time of revision app requests.
+|`configuration_name`, `namespace_name`, `pod_name`, `response_code`, `response_code_class`, `revision_name`, `service_name`
+|Milliseconds
+
+|`revision_queue_depth`
```

Comment:
@abrennan89 thanks for putting this together! Queue proxy metrics need a small fix as the two last metrics are mixed together:
![image](https://user-images.githubusercontent.com/7945591/123650239-0ee8c680-d833-11eb-84dc-a35ad9e60634.png).
You can compare with https://knative.dev/docs/serving/metrics/#queue-proxy

Reply:
Great catch @skonto - I had missed a couple of `Type` columns for the last two metrics 🤦🏻 should be fixed now! 🙂